### PR TITLE
If the database config comes from a URL, ensure that the database nam…

### PR DESCRIPTION
**Pull request by rtfjr86**
…e is not set.

### CODE REVIEW

**Comment by Ocramius** [comment]
@rtfjr86 please add a test scenario checking what happens if the given values collide

File: Command/DropDatabaseDoctrineCommand.php [hunk 1]
```diff
@@ -92,6 +92,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         // Reopen connection without database name set
         // as some vendors do not allow dropping the database connected to.
         $connection->close();
+
+        // If the database config comes from a URL,
+        // ensure that the database name is not set.
+        if (isset($params['url'])) {
+            $params['url'] = str_replace('/'.basename($params['url']), '', $params['url']);
```
Comment (Ocramius) [inline Command/DropDatabaseDoctrineCommand.php]: Should probably throw an exception instead

**Comment by Ocramius** [comment]
Wouldn't mock it at all, but I see that the Travis setup doesn't have a running server. @kimhemsoe any suggestions here?

**Comment by kimhemsoe** [comment]
Can you do the test with sqlite? Noticed that Ryan have been doing that in some of the other tests.

**Pull request closed without merging**